autoapi/extension.py

```diff
@@ -124,4 +126,5 @@ def setup(app):
     app.add_config_value('autoapi_keep_files', False, 'html')
     app.add_config_value('autoapi_add_toctree_entry', True, 'html')
     app.add_config_value('autoapi_template_dir', [], 'html')
+    app.add_config_value('autoapi_url_root', URL_ROOT, 'html')
```

Comment:
It wasn't clear based on the docs, and it seemed it could be an absolute path based on tests. If the intended use of `autoapi_root` was as a path name inside the local path, we should clarify the docs and rework this setting.